README.md

```diff
@@ -16,6 +16,23 @@ you might also like
 
 [Upcase]: https://upcase.com?utm_source=til
 
+## Contributing a TIL
+
+TIL's should be a focused and concise piece of knowledge that's interesting and
```

Comment:
I'm not sure. There's a lot of overlap of the two and I'd like to see where "interesting" would take us. Anyone else have thoughts on this?